MSAL/src/MSALPublicClientApplication.m

```diff
@@ -1260,7 +1279,7 @@ - (void)signoutWithAccount:(nonnull MSALAccount *)account
 {
     __auto_type block = ^(BOOL result, NSError *msidError, id<MSIDRequestContext> context)
     {
-        NSError *msalError = [MSALErrorConverter msalErrorFromMsidError:msidError classifyErrors:YES msalOauth2Provider:self.msalOauth2Provider correlationId:context.correlationId];
+        NSError *msalError = [MSALErrorConverter msalErrorFromMsidError:msidError classifyErrors:YES msalOauth2Provider:self.msalOauth2Provider correlationId:context.correlationId authScheme:nil popManager:nil];
```

Comment:
can we make MSALErrorConverter method with authScheme and popManager an additional one? That way we wouldn't need to pass  authScheme:nil popManager:nil everywhere.

Reply:
This is little tricky as this method internally calls resultWithTokenResult which now requires authScheme to determine what kind of access token it needs to return (signed or unsigned). I would prefer taking it in a separate PR as it is an internal method which we can change later.